src/components/AddDoctor.jsx

```diff
@@ -0,0 +1,190 @@
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { NavLink } from 'react-router-dom';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faCircleArrowLeft } from '@fortawesome/free-solid-svg-icons';
+import { createDoctor } from '../redux/doctorsSlice';
+
+function AddDoctor() {
```

Comment:
- [ ]  Please, kindly refactor this and all other functions in your project to ES6 arrow functions, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const AddDoctor= () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered a non-best practice.

You can do a search in Vscode for the "function" word, and correct as needed.  Shouldn't take you more than 5-10 min so don't worry (I found 8):

![image](https://github.com/CaptainBawa/DocBook-Front-End/assets/57421823/397bfaae-0195-4feb-8db1-8306598c03ae)


That way, your code will meet the latest standards. 💪